### PR TITLE
[PWGUD] Added new variables/cuts + adopted to SG producer

### PR DESCRIPTION
**Pull request by amatyja**
[PWG-UD] Added new variables/cuts + adopted to SG producer

### CODE REVIEW

File: PWGUD/Tasks/upcTauTau13topo.cxx [hunk 1]
```diff
@@ -247,21 +575,22 @@ struct TauTau13topo {
...
     else
       delta -= p1.Phi();
     if (delta < 0)
-      delta += TMath::TwoPi();
-    if (delta > TMath::Pi())
-      delta = TMath::TwoPi() - delta;
+      delta += o2::constants::math::TwoPI;
+    if (delta > o2::constants::math::PI)
+      delta = o2::constants::math::TwoPI - delta;
     return delta;
```
Comment (vkucera) [inline PWGUD/Tasks/upcTauTau13topo.cxx:587]: What are you trying to do here?

Comment (amatyja) [reply PWGUD/Tasks/upcTauTau13topo.cxx:587]: I want to calculate delta phi in a range [0,pi]

Comment (amatyja) [reply PWGUD/Tasks/upcTauTau13topo.cxx:587]: If there is an easier way to do it, let me know.

Comment (vkucera) [reply PWGUD/Tasks/upcTauTau13topo.cxx:587]: `RecoDecay::constrainAngle`.

Comment (rolavick) [reply PWGUD/Tasks/upcTauTau13topo.cxx:587]: Will be resolved in the next occasion

**Review by rolavick — APPROVED** [review]

**Pull request merged**